.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# Gabriel as default owner
+*   @gabalafou
+
+# Noa as default reviewer for Labs blogs and assets
+# Tania/Ralf/others to be manually added as appropriate
+/apps/labs/posts   @noatamir
+/apps/labs/public/posts   @noatamir
```

Comment:
This should be the three of us - and whoever gets first to it will review
it is pretty much what we used to do on our prev site

Reply:
Added, and adjusted spacing to align